### PR TITLE
Dat 2831

### PR DESCRIPTION
**Pull request by jacekjursza**

### CODE REVIEW

**Comment by idradm** [comment]
can't we just ignore empty check in set nodes, and then correct isEmpty to handle such use case?

**Comment by jacekjursza** [comment]
@idradm it's not so simple, since we need to change behavior of data nodes, not set nodes themselves. So we need information about parent node inside data nodes (and preferably in all other nodes too). Also, this solution limits 'ignore' behavior only to data tags inside comparison tag (which allows in future use <set> tags in different context).

File: extensions/wikia/PortableInfobox/services/Parser/Nodes/Node.php [hunk 1]
```diff
@@ -39,7 +58,7 @@ public function setExternalParser( ExternalParser $externalParser ) {
 	}
 
 	public function getType() {
-		return $this->xmlNode->getName();
+		return strtolower( $this->xmlNode->getName() );
```
Comment (SebastianMarzjan) [inline extensions/wikia/PortableInfobox/services/Parser/Nodes/Node.php]: This could use a comment on why strtolower is needed

**Comment by SebastianMarzjan** [comment]
One small request for comment, other than that LGTM

**Comment by SebastianMarzjan** [comment]
:+1:

**Pull request merged**